library/base/src/main/java/org/cru/godtools/base/Settings.java

```diff
@@ -65,7 +65,8 @@ public boolean isFeatureDiscovered(@NonNull final String feature) {
             boolean changed = false;
             switch (feature) {
                 case FEATURE_TUTORIAL_ONBOARDING:
-                    if (getFirstLaunchVersion() <= VERSION_5_1_4) {
+                    if (getFirstLaunchVersion() <= VERSION_5_1_4 ||
+                            Locale.getDefault().getLanguage().startsWith("en")) {
```

Comment:
this test is backwards, it should be `!Locale.getDefault().getLanguage().startsWith("en")`

Reply:
Great Catch I missed it in the change.